tools/accuracy_checker/openvino/tools/accuracy_checker/adapters/background_matting.py

```diff
@@ -0,0 +1,74 @@
+"""
+Copyright (c) 2018-2022 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import cv2
+import numpy as np
+
+from ..adapters import Adapter
+from ..config import StringField
+from ..representation import BackgroundMattingPrediction
+
+
+class ImageBackgroundMattingAdapter(Adapter):
+    __provider__ = 'background_matting_with_pha_and_fgr'
+
+    def process(self, raw, identifiers, frame_meta):
+        result = []
+        frame_meta = frame_meta or [] * len(identifiers)
+        raw_outputs = self._extract_predictions(raw, frame_meta)
+        pha = raw_outputs[self.pha]
```

Comment:
please add outputs verification step (required for matching output tensor and nodes in OV api 2.0)

Reply:
Added.